packages/runner-docker/.gitignore

```diff
@@ -0,0 +1,2 @@
+.dd_key
+.do_key
```

Comment:
these were from when I was testing digital ocean and data dog. Should be removed now.